core/domain/image_services_test.py

```diff
@@ -0,0 +1,128 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for methods in the image_services."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import io
+import os
+
+from PIL import Image
+from constants import constants
+from core.domain import image_services
+from core.tests import test_utils
+import feconf
+import python_utils
+
+
+class ImageServicesUnitTests(test_utils.GenericTestBase):
+    """Tests for image_services."""
+
+    TEST_IMAGE_WIDTH = 3000
+    TEST_IMAGE_HEIGHT = 2092
+
+    def setUp(self):
+        super(ImageServicesUnitTests, self).setUp()
+        with python_utils.open_file(
+            os.path.join(feconf.TESTS_DATA_DIR, 'dummyLargeImage.jpg'),
+            mode='rb', encoding=None) as f:
+            self.jpeg_raw_image = f.read()
+        with python_utils.open_file(
+            os.path.join(feconf.TESTS_DATA_DIR, 'img.png'),
+            mode='rb', encoding=None) as f:
+            self.png_raw_image = f.read()
+
+    def test_image_dimensions_outputs_correctly(self):
+        height, width = (
+            image_services.get_image_dimensions(self.jpeg_raw_image))
+        self.assertEqual(self.TEST_IMAGE_HEIGHT, height)
+        self.assertEqual(self.TEST_IMAGE_WIDTH, width)
+
+    def test_dev_mode_returns_regular_image(self):
```

Comment:
This is confusing because there is no reference in the test to "dev mode". Also not clear what a "regular image" is.